diff --git a/config b/config
index 9589f4f..7775dcc 100755
--- a/config
+++ b/config
@@ -527,7 +527,7 @@ else
    echo "Type 'ls /opt/pgi/osx86-64/*/lib/*       to look for libblas.a from PGI"
 endif
 echo " "
-echo -n "Enter your choice of 'mkl' or 'atlas' or 'acml' or 'pgiblas' or 'none': "
+echo -n "Enter your choice of 'mkl' or 'atlas' or 'acml' or 'pgiblas' or 'openblas' or 'none': "
 set GMS_MATHLIB=$<
 #
 switch ($GMS_MATHLIB)
@@ -750,6 +750,18 @@ switch ($GMS_MATHLIB)
       endif
       set GMS_MATHLIB_PATH=$pgipath
       breaksw
+   case openblas:
+      badopenblas:
+      echo -n "Enter full path to OpenBLAS libraries (without 'lib' subdirectory): "
+      set openblaspath=$<
+      if (!(-d $openblaspath)) then
+         echo " "
+         echo "The directory $openblaspath does not exist."
+         echo " "
+         goto badopenblas
+      endif
+      set GMS_MATHLIB_PATH=$openblaspath/lib
+      breaksw
    default:
       echo "You didn't provide a valid math library name, try again."
       echo " "
diff --git a/lked b/lked
index 8be5dff..3ce66f4 100755
--- a/lked
+++ b/lked
@@ -441,6 +441,11 @@ if ($TARGET == linux32) then
       set BLAS=' '
       breaksw
 
+   case openblas:
+      set MATHLIBS="$GMS_MATHLIB_PATH/libopenblas.a -lpthread -fopenmp"
+      set BLAS=' '
+      breaksw
+
    case none:
    default:
       echo "Warning.  No math library was found, you should install one."
@@ -639,6 +644,11 @@ if ($TARGET == linux64) then
       set BLAS=' '
       breaksw
 
+   case openblas:
+      set MATHLIBS="$GMS_MATHLIB_PATH/libopenblas.a -lpthread -fopenmp"
+      set BLAS=' '
+      breaksw
+
    case none:
    default:
       echo "Warning.  No math library was found, you should install one."
